dominion/dominion_ui/src/components/Hero/Hero.js

```diff
@@ -34,14 +34,43 @@ const styles = theme => ({
       marginTop: 0
     }
   },
-  text: {
+  title: {
     color: 'white',
     width: '90%',
     [theme.breakpoints.down('sm')]: {
       fontSize: '3rem',
       width: '60%'
     }
   },
+  titleFontSmall: {
+    fontSize: '3.125em'
+  },
+  titleWordBreak: {
+    width: 'min-content'
+  },
+  detail: {
+    color: 'white',
+    fontFamily: 'Lora',
+    fontSize: '1.875em',
+    fontWeight: 'normal',
+    fontStyle: 'normal',
+    fontStretch: 'normal',
+    lineHeight: 'normal',
+    letterSpacing: '0.056em'
+  },
+  detailFontSmall: {
+    fontSize: '1.25em'
+  },
+  detailLabel: {
+    color: '#8d8d8c',
+    fontFamily: 'Montserrat',
+    fontSize: '0.688em',
+    fontWeight: 500,
+    fontStyle: 'normal',
+    fontStretch: 'normal',
+    lineHeight: 2.09,
+    letterSpacing: 'normal'
+  },
```

Comment:
You're setting font explicitly here (on `detail`, `detailLabel`, etc.). These should come directly from the theme. If you want to override theming, say use `h3` component but `body1` style say, then:
```jsx
<Typography variant="body1" component="h3">...</Typography>
```
should do the trick.